### ISSUE TITLE
Cannot scroll/right-click menu cuts off for Equipment on NPC sheet

### ISSUE THREAD

**Issue by Arctanis** [Mezryss/FVTT-Genesys#165]
When weapon/armor/gear is added to the Equipment section of an NPC actor sheet, the right-click menu is cut off for at least the first 3 items making it difficult to edit and impossible to delete off the sheet. Also cannot scroll further down to reveal other options.

**Comment by Assembling-Kings** [comment]
Yeah, I've noticed this too and it requires I render the right-click menus differently. For now what I do is I resize the sheet and make it longer, which gives me some extra space at the bottom, but this won't work if the sheet is already pretty big.

**Comment by Arctanis** [comment]
Thanks for the workaround

**Comment by Assembling-Kings** [comment]
The current behavior is that the sheet contents expand to accommodate the context menu. Although the current behavior is not pretty or ideal I'll close this ticket as this is no longer an issue.
